server/database/config/config.js

```diff
@@ -1,18 +1,18 @@
-require('dotenv').config()
+require('dotenv').config();
 
 module.exports = {
   development: {
     username: process.env.DATABASE_USER,
     password: process.env.DATABASE_PASSWORD,
     database: process.env.DATABASE,
-    host: "127.0.0.1",
-    dialect: "postgres",
+    host: '127.0.0.1',
```

Comment:
@MaharjanR I think @shah62 is using a different linter / beautifier than you - we should get on the same page so that both of you have the same formatted code